libdrgn/python/program.c

```diff
@@ -238,10 +239,13 @@ static void drgnpy_end_blocking(struct drgn_program *prog, void *arg, void *stat
 static Program *Program_new(PyTypeObject *subtype, PyObject *args,
 			    PyObject *kwds)
 {
-	static char *keywords[] = { "platform", NULL };
+	static char *keywords[] = { "platform", "vmcoreinfo", NULL };
 	PyObject *platform_obj = NULL;
-	if (!PyArg_ParseTupleAndKeywords(args, kwds, "|O:Program", keywords,
-					 &platform_obj))
+	const char *vmcoreinfo = NULL;
+	size_t vmcoreinfo_size;
```

Comment:
This should technically be `Py_ssize_t`:

```suggestion
	Py_ssize_t vmcoreinfo_size;
```